File: Atividades/Unid-II/Lista01Java/Ex3/src/main/java/home03/Nota.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package home03;

import java.util.ArrayList;

/**
 *
 * @author mathe
 */
public class Nota {
    public boolean verificarAprovado(double n1, double n2, double n3) {
        double M = (n1 + n2) / 2;
        
        if (M >= 7.0) {
            return true;
        } else if (M > 3 && M < 7) {
            if ((M + n3) / 2 >= 5) {
                return true;
            }
        }
        
        return false;
    }
}

File: Atividades-SistemasDistribuidos/Unidade-II/2-Lista01PythonIterativo/Ex3/home03/Nota.py
class Nota:
    def verificarAprovado(n1, n2, n3):
        M = float(n1 + n2) / 2

        if (M >= 7.0):
            return True
        elif (M > 3.0 and M < 7.0):
            if ((M + n3) / 2 >= 5.0):
                return True

        return False